tests/test_behav.py

```diff
@@ -641,3 +645,90 @@ def test_action_value_with_chance_player_action(game: gbt.Game, rational_flag: b
     chance_action = game.players.chance.infosets[0].actions[0]
     with pytest.raises(ValueError):
         game.mixed_behavior_profile(rational=rational_flag).action_value(chance_action)
+
+
+@pytest.mark.parametrize(
+    "game,rational_flag,data",
+    [(games.create_mixed_behav_game(), True,
+     [[[gbt.Rational("0/1"), gbt.Rational("1/1")]],
+      [[gbt.Rational("1/1"), gbt.Rational("0/1")]],
+      [[gbt.Rational("1/1"), gbt.Rational("0/1")]]]),
+     (games.create_mixed_behav_game(), True, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), True, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]]),
+     (games.create_mixed_behav_game(), False, [[[0, 1]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), False, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), False, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]])
+     ]
+)
+def test_specific_profile(game: gbt.Game, rational_flag: bool,
+                          data: typing.Union[gbt.Rational, gbt.Decimal]):
+    """Test that the mixed behavior profile is initialized from a specific distribution
+    for each player over his actions.
+    """
+    profile = game.mixed_behavior_profile(rational=rational_flag, data=data)
+    for (player, behavior) in zip(game.players, data):
+        assert profile[player] == behavior
+
+
+@pytest.mark.parametrize(
+    "game,rational_flag,data",
+    [(games.create_mixed_behav_game(), True,
+      [[[gbt.Rational("0/1"), gbt.Rational("1/1"), gbt.Rational("0/1")]],
```

Comment:
Mixed profiles accept anything that can be interpreted as a number as an argument.  Therefore you do not need `gbt.Rational("0/1")` - you can just write `"0/1"`, and so on.  But for whole numbers you can just do 0 and 1.